Formula/seqan@3.rb

```diff
@@ -15,15 +15,25 @@ class SeqanAT3 < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "doxygen" => :build
   depends_on "xz" => :build
+  depends_on "gcc@8"
 
-  def install
-    system "cmake", "test/documentation/"
-    system "make", "doc_usr"
+  # requires c++17 and concepts
+  fails_with :clang do
+    cause "seqan3 requires concepts and c++17 support"
+  end
```

Comment:
Version 3.0.1 worked with clang. Did a patch release 3.0.2 (or z release) drop support for CLang? That's a big change to make in a patch release.

Reply:
Actually, clang was never supported due to missing concepts support. But our build system has changed, such that we don't need to perform the manual install steps in the formula anymore. But now the cmake script makes sure that all dependencies are met, and this is not possible with the clang compiler (actually never was). So this is rather more accurate now.